src/components/Charts/GrowthRateChart.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import Highcharts, { dateFormat } from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import { GrowthRateChartWrapper } from './GrowthRateChart.style';
+
+// Required for the area-range chart
+require('highcharts/highcharts-more')(Highcharts);
```

Comment:
**NOTE**: We should take a look at the bundle size before merging, if `highcharts/highcharts-more` is too heavy we can re-write the area range chart as stacked area charts and make the lower area invisible (a bit hacky, but should work)